prdoc/schema_user.json

```
@@ -125,10 +124,16 @@
           "name": {
             "type": "string"
           },
+          "bump": {
+            "$ref": "#/$defs/semver_bump"
+          },
           "note": {
             "type": "string"
           }
-        }
+        },
+        "required": [
+          "name"
+        ]
```

Bump also needs to be required if we are to follow SemVer in our new release process with the stable/unstable branches.
```suggestion
        "required": [
          "name",
          "bump"
        ]
```

---

As first step i wanted to make it optional, so we can try it out on a few crates ourselves to ensure that all the tooling/CI checks works.  
If we start with it being required, then many devs will put wrong stuff in there accidentally and we dont notice.

---

Ok, as long as we remember to make it required before we cut the next release yah?